### PR TITLE
feat/(CUST-CPC-1068): Show all customers

### PR DESCRIPTION
**Pull request by AndrewBadIdea**
JIRA: https://champlainsaintlambert.atlassian.net/browse/CPC-1068
## Context:
This ticket is about being able, as an Admin, to view every customers (owners) and their personal informations in a page 
## Changes
* Created a new endpoint to get all owners in the OwnerController
* Created the api call method to the new endpoint (getAllOwners.ts)
* Created a page where you are able to see all the customers
* Adding the routing for getting all the customers in "router.tsx"
* Made a test for the get all to verify the users are Admin
## Before and After UI (Required for UI-impacting PRs)
getallcustomers

### CODE REVIEW

**Review by Justin222993 — APPROVED** [review]
It is perfect

File: petclinic-frontend/src/features/customers/api/getAllOwners.ts
Comment (NotZagreus) [inline petclinic-frontend/src/features/customers/api/getAllOwners.ts:1]: use the new endpoint we created

**Review by flyindonut — DISMISSED** [review]
Good adjustments to use EventSource to handle the text/event-stream format. Looks good to me!

**Review by flyindonut — APPROVED** [review]

**Review by Jesse-Bourassa — APPROVED** [review]

**Review by Justin222993 — APPROVED** [review]
Everything looks good, has been changed to correct endpoint

**Comment by cgerard321** [comment]
Why did you lose the navbar?

**Pull request merged**